ush/sample_config_AQM/config.yaml_nco_realtime_aqmna13km_wcoss2

```diff
@@ -0,0 +1,102 @@
+metadata:
+  description: config for Online-CMAQ, AQM_NA_13km, real-time, NCO mode on WCOSS2
+user:
+  RUN_ENVIR: nco
+  MACHINE: wcoss2
+  ACCOUNT: AQM-DEV
+workflow:
+  USE_CRON_TO_RELAUNCH: true
+  CRON_RELAUNCH_INTVL_MNTS: 3
+#  EXPT_BASEDIR: /lfs/h2/emc/lam/noscrub/chan-hoo.jeon/expt_dirs
+  EXPT_SUBDIR: aqm_nco_aqmna13km
+  PREDEF_GRID_NAME: AQM_NA_13km
+  CCPP_PHYS_SUITE: FV3_GFS_v16
+  DATE_FIRST_CYCL: '2023013000'
+  DATE_LAST_CYCL: '2023013018'
+  INCR_CYCL_FREQ: 6
+  FCST_LEN_HRS: -1
+  FCST_LEN_CYCL:
+    - 6
+    - 72
+    - 72
+    - 6
+  PREEXISTING_DIR_METHOD: rename
+  VERBOSE: true
+  COMPILER: intel
+  DIAG_TABLE_TMPL_FN: diag_table_aqm.FV3_GFS_v16
+  FIELD_TABLE_TMPL_FN: field_table_aqm.FV3_GFS_v16
+nco:
+  envir: prod
+  NET: aqm
+  model_ver: v7.0
+  RUN: aqm_nco_aqmna13km
+#  OPSROOT: /lfs/h2/emc/stmp/chan-hoo.jeon/NCO_OUTPUT_aqm
+workflow_switches:
+  RUN_TASK_MAKE_GRID: false
+  RUN_TASK_MAKE_OROG: false
+  RUN_TASK_MAKE_SFC_CLIMO: false
+  RUN_TASK_RUN_POST: false
+task_make_grid:
+  GRID_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/DOMAIN_DATA/AQM_NA_13km
+task_make_orog:
+  OROG_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/DOMAIN_DATA/AQM_NA_13km
+task_make_sfc_climo:
+  SFC_CLIMO_DIR: /lfs/h2/emc/lam/noscrub/RRFS_CMAQ/DOMAIN_DATA/AQM_NA_13km
+task_get_extrn_ics:
+  EXTRN_MDL_NAME_ICS: FV3GFS
+  FV3GFS_FILE_FMT_ICS: netcdf
+  EXTRN_MDL_ICS_OFFSET_HRS: 0
+task_get_extrn_lbcs:
+  EXTRN_MDL_NAME_LBCS: FV3GFS
+  LBC_SPEC_INTVL_HRS: 6
+  FV3GFS_FILE_FMT_LBCS: netcdf
+  EXTRN_MDL_LBCS_OFFSET_HRS: 0
+  WTIME_GET_EXTRN_LBCS: 02:00:00
+task_run_fcst:
+  DT_ATMOS: 180
+  LAYOUT_X: 50
+  LAYOUT_Y: 34
+  BLOCKSIZE: 16
+  RESTART_INTERVAL: 6 18
+  WTIME_RUN_FCST: 04:00:00
+  QUILTING: true
+  PRINT_ESMF: false
+task_run_post:
+  POST_OUTPUT_DOMAIN_NAME: 793
+global:
+  DO_ENSEMBLE: false
+  NUM_ENS_MEMBERS: 2
+  HALO_BLEND: 0
+  DO_REAL_TIME: true
+cpl_aqm_parm:
+  CPL_AQM: true
+  RUN_TASK_AQM_ICS: true
+  RUN_TASK_AQM_LBCS: true
+  RUN_TASK_NEXUS_GFS_SFC: true
+  RUN_TASK_NEXUS_EMISSION: true
+  RUN_TASK_FIRE_EMISSION: true
+  RUN_TASK_POINT_SOURCE: true
+  RUN_TASK_PRE_POST_STAT: true
+  RUN_TASK_POST_STAT_O3: true
+  RUN_TASK_POST_STAT_PM25: true
+  RUN_TASK_BIAS_CORRECTION_O3: true
+  RUN_TASK_BIAS_CORRECTION_PM25: true
+  WTIME_AQM_LBCS: 01:00:00
+  DO_AQM_CHEM_LBCS: true
+  DO_AQM_GEFS_LBCS: true
+  AQM_BIO_FILE: BEIS_RRFScmaq_C775.ncf
+  DO_AQM_DUST: true
+  AQM_DUST_FILE_PREFIX: FENGSHA_p8_10km_inputs
+  AQM_DUST_FILE_SUFFIX: .nc
+  DO_AQM_CANOPY: false
+  AQM_CANOPY_FILE_PREFIX: gfs.t12z.geo
+  AQM_CANOPY_FILE_SUFFIX: .canopy_regrid.nc
+  AQM_FIRE_FILE_PREFIX: Hourly_Emissions_regrid_NA_13km
+  AQM_FIRE_FILE_SUFFIX: _h72.nc
+  AQM_RC_FIRE_FREQUENCY: hourly
+  DO_AQM_PRODUCT: true
+  AQM_LBCS_FILES: am4_bndy.c793.2019<MM>.v1.nc
+  NEXUS_GRID_FN: grid_spec_793.nc
+  NUM_SPLIT_NEXUS: 3
+#  COLDSTART: false
+#  WARMSTART_CYCLE_DIR: /lfs/h2/emc/ptmp/jianping.huang/para/com/aqm/v7.0/aqm.v7.0.c3/2022102118
```

Comment:
Please rename the test files so that they have a `.yaml` extension.

Reply:
Moved.